webroot/styles/app.css

```diff
@@ -204,3 +204,14 @@ header {
 }
 
 /* ************************************************8 */
+
+
+@media screen and (max-width: 860px) {
```

Comment:
you could move all the styles in this group to line 204, and do away with this line.